reference/ingestion-tools/fleet/install-standalone-elastic-agent.md

```diff
@@ -83,20 +115,16 @@ To install and run {{agent}} standalone:
 
     :::{tab-item} macOS
     
-    ::::{tip}
-    You must run this command as the root user because some integrations require root privileges to collect sensitive data.
-    ::::
+    TIP: You must run this command as the root user because some integrations require root privileges to collect sensitive data.
```

Comment:
Why don't you use a real `tip` admonition inside the tab? Otherwise I would remove the word `TIP:` with capital letters from the sentence.